system_test/aest_commands_SUITE.erl

```diff
@@ -38,7 +39,8 @@
 % Please note: this module is part of of the smoke-test target. The combined
 % runtime should be kept below 10 minutes.
 all() -> [
-    epoch_commands
+    commands,
+    deprecated_commands
```

Comment:
This duplication of test adds 5s to run on CI that is negligible to the time of the system smoke tests (600s+).